server/pages/index.jade

```diff
@@ -46,6 +46,7 @@ html(lang=lang, dir=isRTL ? 'rtl' : 'ltr', class=isFluidWidth ? 'is-fluid-width'
 		link(rel='stylesheet', href='//s1.wp.com/i/fonts/merriweather/merriweather.css?v=20160210')
 		link(rel='stylesheet', href='//s1.wp.com/i/noticons/noticons.css?v=20150727')
 		link(rel='stylesheet', href='//s1.wp.com/wp-includes/css/dashicons.css?v=20150727')
+		link(rel='stylesheet', href='//s1.wp.com/wp-content/mu-plugins/wpcom-smileys/wpcom-smileys.css?v=20160330')
```

Comment:
Why is the url different from the [previous one](https://github.com/Automattic/wp-calypso/pull/4385/files)?